product_docs/docs/pem/8/registering_database_server.mdx

```diff
@@ -8,7 +8,153 @@ redirects:
 - /pem/latest/pem_admin/02_registering_server/
 ---
 
-Before you can manage or monitor a server with PEM, you must register the server with PEM and bind an agent. You can bind a server to a remote agent (an agent that resides on a different host). However, if the agent doesn't reside on the same host, it doesn't have access to all of the statistical information about the instance.
+Before you can manage or monitor a server with PEM, you must register the server with PEM and bind the server to a previously-registered PEM agent. 
+You can bind a server to a remote agent (an agent that resides on a different host). However, if the agent doesn't reside on the same host, it doesn't have access to all of the statistical information about the instance.
+
+## How to register a Postgres server
+There are three methods you can use to register a server.
+
+- You can use the pemworker utility to register a server from the command line on the host where the agent you wish to bind is located.
+- You can use automatic server discovery in the PEM web application to automatically register Postgres servers located on the same host as an agent.
+- You can manually register a Postgres server by adding all the necessary details in the PEM web application.
+
+All three methods are detailed below.
+
+## Using the pemworker utility to register a server
+
+You can use the pemworker utility to register a server for monitoring by the PEM server or to unregister a database server. During registration, the pemworker utility binds the new server to the agent that resides on the system from which you invoked the registration command. To register a server:
```

Comment:
```suggestion
Use the pemworker utility to register a Postgres server to monitor using the PEM server or to unregister a Postgres server. During registration, the pemworker utility binds the new server to the agent on the system from which you invoked the registration command. To register a server:
```

Reply:
I know this is 'house style' thing. But to me changing from stating that you _can_ do something to _instructing_ users to do it harms clarity. This page describes three alternative methods to register servers. If it sounds like we are instructing users to do all of them, I think that will cause confusion.

Reply:
Humm, I see what you mean. If we replace this, "There are three methods you can use to register a server?" with "There are three methods to register a server, use any method to register a server", will it avoid confusion?

Reply:
If the user keeps that context in their head whilst reading this long document, then maybe. But it's hard to see how this can be taken as anything other than an instruction to use pemworker.

> Use the pemworker utility to register a Postgres server to monitor using the PEM server or to unregister a Postgres server.